internal/command/deploy/machines_launchinput_test.go

```diff
@@ -202,7 +202,7 @@ func Test_launchInputForUpdate_keepUnmanagedFields(t *testing.T) {
 	assert.Equal(t, "ord", li.Region)
 	assert.Equal(t, "24/7", li.Config.Schedule)
 	assert.Equal(t, true, li.Config.AutoDestroy)
-	assert.Equal(t, api.MachineRestart{Policy: api.MachineRestartPolicyNo}, li.Config.Restart)
```

Comment:
@dangra  I feel like this test is counterintuitive to what I'm trying to do. What happens if you don't send a Restart policy to flaps? Does it automatically generate one?

Reply:
> What happens if you don't send a Restart policy to flaps? Does it automatically generate one?

Yes, it defaults to "no" restart policy.

> I feel like this test is counterintuitive to what I'm trying to do.

That's true, the reason is that now the restart policy is set from fly.toml so it shouldn't preserve what was previously set for the machine with `fly m update`